CHANGELOG.md

```diff
@@ -74,7 +74,7 @@
 - Remove language from the form validation documentation that doesn't align with the updated guidelines ([#986](https://github.com/opensearch-project/oui/pull/986))
 
 ### 🛠 Maintenance
-
+- Made changes to ```pre-commit``` hook to only use ```yarn lint``` instead of running ```test-staged``` script ([#1072](https://github.com/opensearch-project/oui/pull/1072))
```

Comment:
```suggestion

- Made changes to `pre-commit` hook to only use `lint` instead of running `test-staged` script ([#1072](https://github.com/opensearch-project/oui/pull/1072))
```

Few things:
 1. Would like an empty line before this one to maintain consistency
 2. The code segments only need a single backtick
 3. It should be `lint` instead of `yarn lint`